evals/README.md

```diff
@@ -0,0 +1,82 @@
+
+### 1. Translation (`translation`)
+
+-   **Objective**: To evaluate the model's ability to translate text both to and from a target language.
+-   **Prompt Structure**: A direct, zero-shot English instruction.
+    ```
+    Translate the following text to the {target_language_name} language; use the {script} script; reply only with the translation:
+
+    {original_sentence}
+    ```
+
+### 2. Classification (`classification`)
+
+-   **Objective**: To evaluate the model's ability to classify a paragraph of text into one of five topics.
+-   **Prompt Structure**: A direct, zero-shot English instruction providing the available topics.
+    ```
+    Classify the following text into one of these topics: {topic1}, {topic2}, {topic3}, {topic4}, {topic5}.
+    Reply with only the topic name.
+
+    Text:
+    {paragraph_in_target_language}
+    ```
+
+### 3. Question Answering (`mmlu`, `arc`, `truthfulqa`)
+
+-   **Objective**: To evaluate the model's knowledge and reasoning abilities on multiple-choice questions.
+-   **Prompt Structure**: A zero-shot English instruction combined with a "reasoning scratchpad" format.
+    ```
+    Solve the following multiple choice question. Reason step-by-step and then write the final answer as a single letter.
+
+    Response format: <reasoning> #### <letter>
+
+    ---
+
+    {question_and_choices_in_target_language}
+    ```
+
+### 4. Math Word Problems (`mgsm`)
+
+-   **Objective**: To evaluate the model's ability to solve mathematical reasoning problems.
+-   **Prompt Structure**: Similar to the QA tasks, this uses a zero-shot English instruction with a reasoning scratchpad, but asks for a number as the final answer.
+    ```
+    Solve the following math problem. Reason step-by-step and then write the final answer as a number.
+
+    Response format: <reasoning> #### <number>
+
+    ---
+
+    {math_problem_in_target_language}
+    ```
+
+---
+
+## Advantages and Disadvantages of this Methodology
+
+### Advantages
+
+-   **Fairness and Control**: By using standardized English prompts, we eliminate the quality of prompt translation as a confounding variable, leading to a fairer comparison between models.
+-   **Robustness**: This approach directly tests a model's cross-lingual instruction-following capabilities, which is a key measure of its multilingual prowess.
+-   **Simplicity and Maintainability**: The zero-shot approach significantly simplifies the codebase, making it easier to maintain and extend.
+
+### Disadvantages
+
+-   **Brittleness of Response Parsing**: The evaluation of QA and Math tasks is highly dependent on the model's ability to perfectly adhere to the `#### <answer>` format. Models that produce correct reasoning but fail to follow the format will be unfairly penalized.
+-   **Potential for Cross-Lingual Confusion**: Less capable models may struggle with instructions in one language and content in another, which could impact their performance. 
```

Comment:
Nice!! 🙏
I wonder whether we should add something to make clear that the models should reason in the respective language. [ignore this for review anyway]